jupyter-lab/{{cookiecutter.__repo_slug}}/ods.yaml

```diff
@@ -1,27 +1,27 @@
-branchToEnvironmentMapping:
-- branch: '*'
-  environment: dev
-
-environments:
-- name: dev
-  namespace: {{cookiecutter.__project_slug}}-dev
-  stage: dev
-
 pipeline:
```

Comment:
Should this be `pipelines`?

Reply:
Same goes for the others

Reply:
If there is just one target namespace, it is not needed. If there are different targets based on different branches, then pipeline triggers would be used, as explained in https://github.com/opendevstack/ods-pipeline/blob/main/docs/ods-configuration.adoc#configuring-pipeline-triggers. Would you add that as an example here? I thought to keep it super simple initially.